.github/workflows/kibana-reports-test-and-build-workflow.yml

```diff
@@ -32,9 +32,9 @@ jobs:
         run: |
```

Comment:
This step is missing in the release flow

Reply:
Added, although I think we only need to test in CI, shouldn't need to test in CD?

Reply:
it may be a requirement from infra team, guess all the plugins are doing this way